doc/sources/guide/licensing.rst

```diff
@@ -0,0 +1,137 @@
+Package licensing
+=================
+
+.. warning:: This is not a lawyer consulted guide! Kivy organisation, authors
+   or contributors to this guide take no responsibility for any lack of
+   information, misleading information presented here or any actions based on
+   this guide. The guide is merely informative and is meant to protect
+   inexperienced users.
+
+Your code alone may not require including licensing info and copyright notices
+of other used software, but binaries are something else. When a binary (.exe,
+.app, .apk, ...) is created it includes Kivy, its dependencies and other
+packages that your application uses. Some of them are licensed in a way they
+require including a copyright notice somewhere in your app (or more). Before
+distributing any of the binaries, please **check all the created files** that
+don't belong to your source (.dll, .pyd, .so, ...) and include approperiate
+copyright notices if required by the license the files belong to. This way you
+should satisfy licensing requirements of the Kivy deps.
+
+Dependencies
+------------
+
+All of the dependencies will jump out at least partially on each platform Kivy
+supports, therefore you need to comply to their licenses, which mostly requires
+only pasting a copyright notice to your app and not pretending you wrote the
+code.
+
+.. |mixer| replace:: SDL_mixer has them
+.. _mixer: http://hg.libsdl.org/SDL_mixer/file/efa81a285f22/VisualC/external/lib/x86
+
+* `docutils <https://sourceforge.net/p/docutils/code/HEAD/tree/trunk/docutils/COPYING.txt>`_
+* `pygments <https://bitbucket.org/birkenfeld/pygments-main/src/a042025b350cd9c9461f7385d9ba0f13cdb01bb9/LICENSE>`_
+* `sdl2 <https://www.libsdl.org/license.php>`_
+* `glew <http://glew.sourceforge.net/glew.txt>`_
+* `gstreamer <https://github.com/GStreamer/gstreamer/blob/master/COPYING>`_
+  (if used)
+* image & audio libraries(e.g. |mixer|_)
+
+You'll probably need to check image and audio libraries manually (mostly
+begin with ``lib``). The ``LICENSE*`` files that belong to them should be
```

Comment:
nope, those are copied only because of the kivy.deps hook, because SDL2 (or Kivy in kivy.deps.sdl2) has the LICENSE* files distributed together with binaries. If you have an access to Win, check wheel installation and `<python>/share/sdl2/bin` folder